controller/internal/enforcer/nfqdatapath/datapath_tcp.go

```diff
@@ -4,14 +4,17 @@ package nfqdatapath
 import (
 	"errors"
 	"fmt"
+	"reflect"
 	"strconv"
```

Comment:
no reflect on the data path even for logs

Reply:
removed